ci-operator/step-registry/cucushift/installer/rehearse/aws/ipi/disconnected/private/by-oc-mirror/sno/cucushift-installer-rehearse-aws-ipi-disconnected-private-by-oc-mirror-sno-workflow.yaml

```diff
@@ -0,0 +1,11 @@
+workflow:
+  as: cucushift-installer-rehearse-aws-ipi-disconnected-private-by-oc-mirror-sno
```

Comment:
Can you help replace the install workflow with this new one on https://github.com/openshift/release/blob/master/ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.15__amd64-stable.yaml#L119 , so that we can run pr-rehearse job to validate if the new install workflow work or not.

Reply:
@jianlinliu  we can't use the workflow as you said , because we will hit ci/prow/ci-operator-config and ci/prow/ci-operator-registry job failed

Reply:
so I just restore the update.

Reply:
> because we will hit ci/prow/ci-operator-config and ci/prow/ci-operator-registry job failed

I do not think so, pls check the detail log of the failed jobs, e.g:
https://prow.ci.openshift.org/view/gs/test-platform-results/pr-logs/pull/openshift_release/46249/pull-ci-openshift-release-master-ci-operator-config/1747535687193726976
```
time="2024-01-17T08:29:21Z" level=fatal msg="failed to parse arguments" error="failed to load registry: name of workflow in file ci-operator/step-registry/cucushift/installer/rehearse/aws/ipi/disconnected/private/by-oc-mirror/sno/cucushift-installer-rehearse-aws-ipi-disconnected-private-by-oc-mirror-sno-workflow.yaml should be cucushift-installer-rehearse-aws-ipi-disconnected-private-by-oc-mirror-sno"
```
I think the log is clear enough, sounds you did not input correct name in the install workflow yaml.